app/models/lab_test.rb

```diff
@@ -13,6 +13,35 @@ class LabTest < ApplicationRecord
                       message: :must_be_nonnegative_and_numeric
                     }
 
+  scope :in_date_range, lambda { |start_date, end_date|
+    scope = all
+    scope = scope.where(created_at: Date.parse(start_date).beginning_of_day..) if start_date.present?
+    scope = scope.where(created_at: ..Date.parse(end_date).end_of_day) if end_date.present?
+    scope
+  }
+
+  scope :by_date, lambda { |date|
+    where(created_at: date.all_day)
+  }
+
+  # Show all records, ordered by newest first
+  scope :latest_records, lambda {
```

Comment:
What do you think about name it `:order_by_latest_records` - will bring transparency and it will be obvious what is going, especially that this is about ordering, not just filtering.